core/db/migrate/20191017121054_add_supported_currencies_to_store.rb

```diff
@@ -0,0 +1,5 @@
+class AddSupportedCurrenciesToStore < ActiveRecord::Migration[6.0]
+  def change
+    add_column :spree_stores, :supported_currencies, :string
```

Comment:
@btolarz let's also set the default value

```suggestion
    add_column :spree_stores, :supported_currencies, :string
    Spree::Store.each do |store|
      store.update_attribute(supported_currencies: store.default_currency)
    end
```

Reply:
done